### ISSUE TITLE
Module '"@uidotdev/usehooks"' has no exported member 'useLocalStorage'.

### ISSUE THREAD

**Issue by piotrkulpinski** [uidotdev/usehooks#230]
Hi,
I'm trying to use the 'useLocalStorage' but it seems like the package doesn't export it properly. 
<img width="815" alt="CleanShot 2023-09-06 at 13 05 11" src="https://github.com/uidotdev/usehooks/assets/358412/872eff9a-79d2-4002-977c-c5ccadba43e2">

All other hooks are working fine. 
I'm using v2.2.0.

Thanks

**Comment by isimmons** [comment]
Yep the type declaration is missing from index.d.ts

I put this in there
```js
export function useLocalStorage<T>(
    key: string,
    initialValue: T
  ): [T , (v: T) => void];
```
It works for this simple test component
```js
import { useEffect } from 'react';
import { useLocalStorage } from '@uidotdev/usehooks';

type LocalStore = Record<string, string> | null;

const FooBar = () => {
  const [store, setStore] = useLocalStorage<LocalStore>('foo', null);

  useEffect(() => {
    setStore({ foo: 'bar' });
  }, [setStore]);

  return <div>{store?.foo}</div>;
};

export default FooBar;
```
But I don't know how this works yet with more complex things . I'm going to try it with the drawing demo from the docs.

UPDATE: It also works with the drawing demo. Just change the generic 
```js
const [drawing, saveDrawing] = useLocalStorage<string | null>(
    'drawing',
    null
  );
```

**Comment by isimmons** [comment]
I see someone already put in a [PR for this](https://github.com/uidotdev/usehooks/pull/228)

**Comment by colinbr96** [comment]
Temporary workaround:
```typescript
declare module "@uidotdev/usehooks" {
  export function useLocalStorage<T>(
    key: string,
    initialValue: T
  ): [T , (v: T) => void];
}
```

**Comment by tylermcginnis** [comment]
Fixed in #228 . Release will go out soon.
